Resources/Prototypes/DeltaV/Entities/Objects/Devices/Syndicate_Gadgets/reinforcement_teleporter.yml

```diff
@@ -0,0 +1,21 @@
+- type: entity
+  parent: ReinforcementRadioSyndicate
+  id: ReinforcementRadioSyndicateNukieMouse
+  name: suspicious moldy cheese
+  description: Moldy cheese with a little worm sticking out of it and a... blinking antenna? Might attract an odd mouse.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Objects/Devices/communication.rsi
+    layers:
+    - state: cheese-radio
+  - type: GhostRole
+    name: Nuclear Mouse Operator Reinforcement
+    description: Someone needs reinforcements. You, a trained mouse, will help them.
+    rules: Normal syndicate antagonist rules apply. Work with whoever called you in, and don't harm them.
```

Comment:
These should be localized.

Reply:
How would I do that?

Reply:
I was able to localize them in the fixed branch https://github.com/DeltaV-Station/Delta-v/pull/498